pkg/workloads/console/runner/runner.go

```diff
@@ -232,11 +228,36 @@ func (c *Runner) Create(ctx context.Context, opts CreateOptions) (*workloadsv1al
 				Hook:       opts.Hook,
 			},
 		)
+		if err != nil {
+			return csl, err
+		}
+
+		return csl, c.exitWithStatus(csl)
```

Comment:
Why add this here, rather than in `Runner.Attach` where it would take effect for both create and attach?

Regardless of what kubectl does I reckon it's a bit confusing to have one subcommand check for the exit status and the other to not, even more so if undocumented!

Reply:
I've moved it, since it doesn't make things any more complicated.